packages/core/src/App/Containers/ResetEmailModal/confirm-email-modal.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { SentEmailModal } from '@deriv/account';
+import { WS } from 'Services';
+import PropTypes from 'prop-types';
+import { connect } from 'Stores/connect';
+
+export const ConfirmEmailModal = ({ onClose, is_open, prevEmail, newEmail, verification_code, pullErrorMessage }) => {
+    const [email_request, setEmailRequest] = React.useState(null);
+    const [is_send_email_modal_open, setIsSendEmailModalOpen] = React.useState(false);
+    const [is_error, set_is_error] = React.useState(false);
+    const [error_message, set_error_message] = React.useState(null);
+    const [isOpen, setIsOpen] = React.useState(is_open);
+
+    const handleSubmit = () => {
+        const api_request = {
+            change_email: 'verify',
+            new_email: newEmail,
+            verification_code,
+        };
+
+        setEmailRequest(prev => ({ ...prev, ...api_request }));
+
+        WS.changeEmail(api_request).then(response => {
+            if (response.error) {
+                setIsOpen(false);
+                onClose(true);
```

Comment:
onClose function is not accepting any param, do we need to pass this true?

Reply:
Yeah. If there is an error, the previous page will pop up